src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3659,6 +3659,12 @@
   <data name="SortPriority" xml:space="preserve">
     <value>Sort priority</value>
   </data>
+  <data name="Quit" xml:space="preserve">
```

Comment:
```suggestion
  <data name="CloseFiles" xml:space="preserve">
```

Reply:
I mistakenly merged your suggestion. IIRC we've decided what to call [before](https://discord.com/channels/725513575971684472/1164222926749442059/1164224058670448753). But I think we can decide again lol.

Reply:
Either one works

Reply:
User would be confused why have to close again. Idk but I often see quit but never close.